app/models/manageiq/providers/container_manager.rb

```diff
@@ -30,5 +32,10 @@ def aggregate_cpu_total_cores(targets = nil)
     def aggregate_memory(targets = nil)
       aggregate_hardware(:computer_systems, :memory_mb, targets)
     end
+
+    def add_default_policies
+      # make sure that add_policy indeed adds a policy profile (aka MiqPolicySet)
+      add_policy(MiqPolicySet.find_by_guid('747a70d0-a8bf-4b63-bdad-1676bc0c5356'))
```

Comment:
@blomquisg Looks like that guid is contained within the seed data here [7558](https://github.com/ManageIQ/manageiq/pull/7558/files#diff-1cc86ebb35ea7f32ab4c185735cd3612R4)

But, I'm also uncomfortable with referencing it with the guid. Maybe you can give it a real name instead using the guid and then do a find_by_name instead. That would make this code much easier to read.